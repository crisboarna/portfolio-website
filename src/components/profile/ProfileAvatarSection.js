import React from 'react';
import ScrollAnimation from 'react-animate-on-scroll';
import Scrollchor from 'react-scrollchor';

import Avatar from '../../media/Avatar.png';
import '../../styles/profile/ProfileAvatarSection.css';

const ProfileAvatarSection = () => {
  return (
    <div className="col-xs-6 col-lg-6">
      <ScrollAnimation animateIn='fadeInLeft' animateOnce={true} offset={50}>
        <img src={Avatar} className='cropShield' alt='Avatar'/>
        <p className='avatarText'>I am afire with desire to sire cloud native fullstack systems.
          <br/>
          Experienced in delivering systems to companies ranging from investment banks to small startups.
          <br/>
          <Scrollchor to='#contact'><span></span>Let's make your dream come true.</Scrollchor>
        </p>
      </ScrollAnimation>
    </div>
  );
};

export default ProfileAvatarSection;