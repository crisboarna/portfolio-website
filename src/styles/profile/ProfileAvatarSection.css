.avatarText {
    color: #616161;
    font-size: 12pt;
    padding-top: 3%;
    font-family: "Raleway";
}

.cropShield {
    width: 250px;
    height: 250px;
    border-top-left-radius: 10% 10%;
    border-top-right-radius: 10% 10%;
    border-bottom-right-radius: 50% 50%;
    border-bottom-left-radius: 50% 50%;
}