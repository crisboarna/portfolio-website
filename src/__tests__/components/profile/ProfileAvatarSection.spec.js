import React from 'react';
import { shallow, mount } from 'enzyme';
import ProfileAvatarSection from '../../../components/profile/ProfileAvatarSection';

describe('ProfileAvatarSection', () => {
  it('renders without crashing', () => {
    const component = shallow(<ProfileAvatarSection />);
    expect(component.exists()).toEqual(true);
  });

  it('compare to snapshot', () => {
    const component = mount(<ProfileAvatarSection/>);
    expect(component).toMatchSnapshot();
  });
});