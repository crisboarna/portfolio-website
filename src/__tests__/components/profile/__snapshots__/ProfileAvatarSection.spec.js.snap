// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileAvatarSection compare to snapshot 1`] = `
<ProfileAvatarSection>
  <div
    className="col-xs-6 col-lg-6"
  >
    <ScrollAnimation
      animateIn="fadeInLeft"
      animateOnce={true}
      delay={0}
      duration={1}
      initiallyVisible={false}
      offset={50}
    >
      <div
        className="animated"
        style={
          Object {
            "animationDuration": "1s",
            "opacity": 0,
          }
        }
      >
        <img
          alt="Avatar"
          className="cropShield"
          src="Avatar.png"
        />
        <p
          className="avatarText"
        >
          I am afire with desire to sire cloud native fullstack systems.
          <br />
          Experienced in delivering systems to companies ranging from investment banks to small startups.
          <br />
          <Scrollchor
            to="#contact"
          >
            <a
              href="#contact"
              onClick={[Function]}
            >
              <span />
              Let's make your dream come true.
            </a>
          </Scrollchor>
        </p>
      </div>
    </ScrollAnimation>
  </div>
</ProfileAvatarSection>
`;
